index.html

```diff
@@ -5,12 +5,29 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles/globals.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.css%22%3E"
+    />
+
     <title>Space Launches</title>
   </head>
 
   <body>
     <main>
-      <h1 class="title">Space Launches</h1>
+      <section class="page-cover">
+        <h1 class="title">
+          Space Launches<span class="sub-title"
```

Comment:
Podemos revisar si poniendo un espacio acá prettier identa bien el codigo?
```suggestion
          Space Launches <span class="sub-title"
```

Reply:
Este comentario no aplica porque sacamos el `span ` del `h1`.